.github/workflows/release-test.yml

```diff
@@ -0,0 +1,55 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+##### README #####
+#
+# This CI action runs the release crates as downloaded from crates.io. It does not block PRs
+# and it is okay if it is flaky from time to time.
+
+name: Release Build
```

Comment:
I don't think this makes much sense. This runs on every push, but the release crates only change once every few months. Also, we'll only realise that we've broken an example *after the release*.

What I'd recommend doing instead:

* create a new workspace in `tutorials/cargo_tests`
  * (why `cargo_tests` and not `cargo` anyway?)
* use the [`patch` key](https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html#the-patch-section) on the workspace level to point all crates to local crates without specifying that in the examples themselves.
* If `tutorials/cargo_tests` itself changes, run a second pass with the patch key removed to test against crates.io

Reply:
Good idea! The catch with using `.cargo/config.toml` is that it's hard to unset the configurations; you need to do it one by one. So instead I created the config file, named it `patch.toml`, and when you want to use it, you pass it with the `--config` CLI option. I made a cargo-make job that applies the patch and hooked it up to run in CI.